src/backend/distributed/utils/citus_copyfuncs.c

```diff
@@ -85,6 +85,7 @@ copyJobInfo(Job *newnode, Job *from)
 	COPY_SCALAR_FIELD(requiresMasterEvaluation);
 	COPY_SCALAR_FIELD(deferredPruning);
 	COPY_NODE_FIELD(partitionKeyValue);
+	COPY_NODE_FIELD(localPlannedStatements);
```

Comment:
Just a thought: we could deliberately exclude it from here.

Reply:
Or even better, make it a SCALAR field instead of a NODE field

Reply:
I think what we have (e.g., `CopyDistributedPlanWithoutCache` ) is better in the sense that any caller who wants that specific behaviour can choose to do so. But doing it here might be confusing and could easily be broken, someone else realizing and trying to fix it.